server/routes/refer.ts

```diff
@@ -6,7 +6,7 @@ import { referPaths } from '../paths'
 import { RouteUtils } from '../utils'
 
 export default function routes(controllers: Controllers, router: Router): Router {
-  const { get, post, put } = RouteUtils.actions(router)
+  const { get, post, put } = RouteUtils.actions(router, { allowedRoles: [ApplicationRoles.ACP_REFERRER] })
```

Comment:
Ah this is a good way to make the file a bit more readable, nice work. I think it might be good to document what overwriting these to make them e.g. open to all users would look like in the commit message - would we pass an empty object here or list all of the roles it's available to?

Reply:
Like this, nice one.

Reply:
I think it would be exactly the same as before, since the previous and this approach should both resolve to `undefined` if the argument isn't passed in

Reply:
(but yeah we should document it - I'll pick it up on Monday unless anyone wants to tidy this up tomorrow)